modules/simple_forms_api/app/form_mappings/vba_21_0845.json.erb

```diff
@@ -83,11 +83,11 @@
   "F[0].#subform[2].International_Telephone_Number_If_Applicable[3]": "<%= form.data['security_answer'] && form.data['security_question'] == 'favoriteTeacherName' ? form.data['security_answer'] : '' %>",
   "F[0].#subform[2].International_Telephone_Number_If_Applicable[4]": "<%= form.data['security_answer'] && form.data['security_question'] == 'fatherMiddleName' ? form.data['security_answer'] : '' %>",
 
-  "F[0].#subform[2].Date_Signed_Year[0]": "<%= Time.current.in_time_zone('America/Chicago').strftime('%Y') %>",
-  "F[0].#subform[2].Date_Signed_Day[0]": "<%= Time.current.in_time_zone('America/Chicago').strftime('%d') %>",
-  "F[0].#subform[2].Date_Signed_Month[0]": "<%= Time.current.in_time_zone('America/Chicago').strftime('%m') %>",
-  
+  "F[0].#subform[2].Date_Signed_Year[0]": "<%= form.signature_date.strftime('%Y') %>",
```

Comment:
I think you may want to use safe navigation here and in other places as well.

`form.signature_date&.strftime` or even `form&.signature_date&.strftime`

Reply:
Excellent idea, fixed! 👍